js/build.js

```diff
@@ -22,6 +22,13 @@
     var $container = $(this);
     var deletedAllSlides = false;
 
+    // Setting container parent max-width to 100% so that the swiper library set correct slide width
+    // To work with old D&D
+    $container.parents('.column-item').css('maxWidth', '100%')
```

Comment:
I don't understand this fix. We've got `$container`, which is the DOM element wrapping the component, which means `$container.parents('.column-item')` must be referring to DOM elements outside of the component.

I don't there's a guarantee that there's a `.column-item` on the page so this fix seems to be dependent on or addressing an issue that is outside of the concern of this component.

So, to clarify, what is `$container.parents('.column-item')` and `$container.parent()` addressing and how does this bugfix work?

Reply:
@tonytlwu 

> I don't there's a guarantee that there's a .column-item on the page so this fix seems to be dependent on or addressing an issue that is outside of the concern of this component.

`.column-item` presents in the DOM only with old D&D and if this class isn't found it means that we need set `max-height` to component direct parent in witch we are containing.

> So, to clarify, what is $container.parents('.column-item') and $container.parent() addressing and how does this bugfix work?

Let us start that `Swiper`, lib that we are using to create a slider, set inline width to each slide according to its container width. That's why we didn't see the slide because it was outside the viewport we can see. Before this fix parents of this component don't has any fixed width and one slide could be bigger than 1000px width and that's why we didn't see it. This fix gives component parent container `max-width` of 100% on with the help of this style library can calculate the right width for each slide, and solve this issue.

Reply:
I think this is possibly occurring due to `.column-item` on the old system (for one of the layouts) and the new system both use flexbox sparingly. This explains why the container outside of the component needs a width/max-width set. However, it still comes back to the fact that the component should not be dependent on assessing what's outside of the component.

@hcarneiro Do you have any recommendations on how to fix this?